### ISSUE TITLE
Grammar error in document: Quickstart

### ISSUE THREAD

**Issue by forrestchang** [kennethreitz/requests#3508]
In section [Binary Response Content](http://docs.python-requests.org/en/master/user/quickstart/#binary-response-content), `r.content` returns an instance of `bytes` while `StringIO` is an in-memory stream for text only. It should use `BytesIO` instead.

```python
from PIL import Image
from io import BytesIO

i = Image.open(BytesIO(r.content))
```

Use original code will raise `TypeError: initial_value must be str or None, not bytes.`

**Comment by Lukasa** [comment]
@forrestchang Thanks, would you like to open a PR to change it?

**Comment by forrestchang** [comment]
@Lukasa Yes, I would like to. I have modified the mistake in `.rst` file. Should I generate it to the HTML file?

**Comment by nateprewitt** [comment]
Hey @forrestchang, there's no need to run the docs build. If you open a PR with the rst changes, read the docs will generate changes on merge.

**Comment by forrestchang** [comment]
@nateprewitt Thanks.

**Comment by Lukasa** [comment]
Resolved by #3511.
